java/arcs/core/host/AbstractArcHost.kt

```diff
@@ -60,6 +65,7 @@ typealias ParticleRegistration = Pair<ParticleIdentifier, ParticleConstructor>
 @ExperimentalCoroutinesApi
 abstract class AbstractArcHost(
     protected val coroutineContext: CoroutineContext = Dispatchers.Default,
+    updateArcHostContextCoroutineContext: CoroutineContext = Dispatchers.Default,
```

Comment:
Is the main goal of adding this parameter so that we could change to a different dispatcher if needed?

Reply:
Yes, like the cl that is waiting for this PR: [cl/324891905](http://cl/324891905)